<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditTicketView</class>
 <widget class="QDialog" name="EditTicketView">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Ticket</string>
  </property>
  <property name="styleSheet">
   <string notr="true">EditTicketView { background: qlineargradient(spread:pad, x1:0.573909, y1:0.693, x2:0, y2:0, stop:0 rgba(176, 227, 100, 255), stop:1 rgba(255, 255, 255, 255)); }</string>
  </property>
  <widget class="QLabel" name="firstNameLabel">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Category:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ticket Number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lastNameLabel">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>130</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Severity:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="description">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>481</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TicketNum">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Subject: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="severityCombo">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>150</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="agentIdInput">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>200</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="closeDate">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>80</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="subject_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QLabel" name="openDate">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Open Date/Time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="userLabel">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>180</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Agent ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Customer Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ticket Details</string>
   </property>
  </widget>
  <widget class="QComboBox" name="categoryCombo">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>40</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Close Date/Time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="cusName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="userError">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>230</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Assigned To:</string>
   </property>
  </widget>
  <widget class="QLabel" name="agentLabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="statusDropDown">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>90</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>70</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>801</width>
     <height>281</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background: url(&quot;:/new/prefix1/loginButtonImage.png&quot;) repeat-x; 
width: 200px;  
padding:10px 0;  
color: palette(shadow); 
text-align:center; 
border:1px solid #608925;  
font-weight:bold;  
text-transform:uppercase;  
border-radius: 30px;  
color:#FFF;  
text-decoration: none;  
opacity:.85;  
}

QPushButton:hover {
border-color:#49671d;  
}


QPushButton:active {  
background-position:  0 -200px;  
padding:11px 0 9px;  
}  </string>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QLabel" name="addCommnetError">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="bodyError">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prevButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background: url(&quot;:/new/prefix1/loginButtonImage.png&quot;) repeat-x; 
width: 200px;  
padding:10px 0;  
color: palette(shadow); 
text-align:center; 
border:1px solid #608925;  
font-weight:bold;  
text-transform:uppercase;  
border-radius: 30px;  
color:#FFF;  
text-decoration: none;  
opacity:.85;  
}

QPushButton:hover {
border-color:#49671d;  
}


QPushButton:active {  
background-position:  0 -200px;  
padding:11px 0 9px;  
}  </string>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>200</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background: url(&quot;:/new/prefix1/loginButtonImage.png&quot;) repeat-x; 
width: 200px;  
padding:10px 0;  
color: palette(shadow); 
text-align:center; 
border:1px solid #608925;  
font-weight:bold;  
text-transform:uppercase;  
border-radius: 30px;  
color:#FFF;  
text-decoration: none;  
opacity:.85;  
}

QPushButton:hover {
border-color:#49671d;  
}


QPushButton:active {  
background-position:  0 -200px;  
padding:11px 0 9px;  
}  </string>
    </property>
    <property name="text">
     <string>Add Comment</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="bodyInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>771</width>
      <height>76</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="authorLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Author:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>651</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background: url(&quot;:/new/prefix1/loginButtonImage.png&quot;) repeat-x; 
width: 200px;  
padding:10px 0;  
color: palette(shadow); 
text-align:center; 
border:1px solid #608925;  
font-weight:bold;  
text-transform:uppercase;  
border-radius: 30px;  
color:#FFF;  
text-decoration: none;  
opacity:.85;  
}

QPushButton:hover {
border-color:#49671d;  
}


QPushButton:active {  
background-position:  0 -200px;  
padding:11px 0 9px;  
}  </string>
    </property>
    <property name="text">
     <string>Save Comment Changes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background: url(&quot;:/new/prefix1/loginButtonImage.png&quot;) repeat-x; 
width: 200px;  
padding:10px 0;  
color: palette(shadow); 
text-align:center; 
border:1px solid #608925;  
font-weight:bold;  
text-transform:uppercase;  
border-radius: 30px;  
color:#FFF;  
text-decoration: none;  
opacity:.85;  
}

QPushButton:hover {
border-color:#49671d;  
}


QPushButton:active {  
background-position:  0 -200px;  
padding:11px 0 9px;  
}  </string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLabel" name="dateLabel">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Date:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Comments</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="noteButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: url(&quot;:/new/prefix1/loginButtonImage.png&quot;) repeat-x; 
width: 200px;  
padding:10px 0;  
color: palette(shadow); 
text-align:center; 
border:1px solid #608925;  
font-weight:bold;  
text-transform:uppercase;  
border-radius: 30px;  
color:#FFF;  
text-decoration: none;  
opacity:.85;  
}

QPushButton:hover {
border-color:#49671d;  
}


QPushButton:active {  
background-position:  0 -200px;  
padding:11px 0 9px;  
}  </string>
   </property>
   <property name="text">
    <string>View Notes</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="saveCancelButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>540</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: url(&quot;:/new/prefix1/loginButtonImage.png&quot;) repeat-x; 
width: 200px;  
padding:10px 0;  
color: palette(shadow); 
text-align:center; 
border:1px solid #608925;  
font-weight:bold;  
text-transform:uppercase;  
border-radius: 30px;  
color:#FFF;  
text-decoration: none;  
opacity:.85;  
}

QPushButton:hover {
border-color:#49671d;  
}


QPushButton:active {  
background-position:  0 -200px;  
padding:11px 0 9px;  
}  </string>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="subject">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
